server/sonar-server/src/main/java/org/sonar/ce/monitoring/CEQueueStatus.java

```diff
@@ -134,5 +84,5 @@
   /**
    * Time spent processing batch reports since startup.
    */
-  long getProcessingTime();
+  long getProcessingTimeInMs();
```

Comment:
This pattern helped me a lot in the past to avoid bugs. I don't think you can compare with java.util.Date#getTime(), which is a central piece of the JDK.